package/tvheadend-app-osmc/files/DEBIAN/postinst

```diff
@@ -41,6 +94,61 @@ if [ "$1" = "configure" ]; then
 		fi
 	fi
 
-	if ischroot; then exit 0; fi
+	# start tvh so that it updates any old user files
+	echo Starting tvheadend to finalise setup for osmc
 	if systemctl is-enabled tvheadend.service >/dev/null; then systemctl start tvheadend.service; fi
+	#systemctl restart tvheadend
+	sleep 5
```

Comment:
You'll have to explain this one a bit better, but here are the two issues:

* 5 seconds doesn't guarantee TVH has started
* We can't guarantee TVH service starts at all

I'd be more inclined to check if OSMC user exists and if so, to recreate the config and move the old one to a .bak, assuming it's just userdata and not channel config etc.